nix/tests.nix

```diff
@@ -112,6 +113,58 @@ let
 
         ${withTmpDb postgresql} "$rootdir/test/memory-tests.sh"
       '';
+
+  coverage =
+    name: postgresql:
+    let
+      covCabalOptions="${devCabalOptions} --enable-coverage --enable-executable-dynamic";
+    in
+    checkedShellScript
+      name
+      ''
+        env="$(cat ${postgrest.env})"
+        export PATH="$env/bin:$PATH"
+
+        rootdir="$(${git}/bin/git rev-parse --show-toplevel)"
+        cd "$rootdir"
+
+        # clean up previous coverage data
+        rm -rf coverage
+        find . -type f -regextype posix-extended -regex ".*\.(tix)" -print0 | xargs -0 rm -f
```

Comment:
Note to self: I think I can simplify the regex. It was more complicated before, but doesn't need to be posix-extended anymore...